### ISSUE TITLE
Allow retraining SAEs

### ISSUE THREAD

**Issue by danbraunai-apollo** [ApolloResearch/e2e_sae#15]
We may want to initialize the SAEs to be some already pre-trained SAEs for stability and ease of training. We'd just add an arg pretrained_sae_path and load up the SAEs there before training.

If we get to the point where our runs are taking 1-2+ hours, we can try allow for training continuation by reading in the number of samples the pretrained SAEs were trained for, and the seed, and optimizer state (these should be saved along with the model along with other misc information), and continue training from where we left off.
